e2e/protractor.conf.js

```diff
@@ -12,7 +12,7 @@ exports.config = {
 // Firefox local setup
   exclude: ['./src/**/download.e2e-spec.ts', "./src/**/print.e2e-spec.ts"],
   capabilities: {
-    browserName: 'firefox',
+    browserName: 'chrome',
```

Comment:
could you uncomment the Chrome local setup section and comment the Firefox local setup bit out? That way people won't have to look into the code to switch between the two.

Reply:
cucumber e2e tests using cucumber.conf.js